eoglearn/datasets/eegeyenet.py

```diff
@@ -54,13 +63,14 @@ def fetch_eegeyenet(subject="EP10", run=1, fetch_dataset_kwargs=None):
     pathlib.Path
         Path to the downloaded file.
     """
+    task = _get_task_from_subject_id(subject)
```

Comment:
Since `get_subject_runs` API now has a parameter `task="DOT"`. We need to actually pass `task="AS"` in cases where the filename is an anti-saccade file, e.g. `fetch_eegeyenet(subject="BZ2")`.

I created little helper function for this. For some reason it feels a little hacky, but it works. 

So I just want to make sure everyone agrees with this approach.

Reply:
Seems fine to me. I think what is not optimal is for the task to be part of the subject ID (you cannot have the same subject doing two tasks), but that was decided by EEGEyeNet, so it is fine to use it I think...